python/pyspark/sql/pandas/conversion.py

```diff
@@ -34,7 +34,7 @@ class PandasConversionMixin(object):
     """
 
     @since(1.3)
-    def toPandas(self):
+    def toPandas(self, self_destruct=False):
```

Comment:
Let's name it as `selfDestruct` to make the naming role consistent.

Reply:
Done.